openwisp_users/multitenancy.py

```diff
@@ -65,6 +65,8 @@ def _edit_form(self, request, form):
         else show everything
         """
         fields = form.base_fields
+        if 'organization' in fields:
+            fields['organization'].empty_label = 'Shared systemwide (no organization)'
```

Comment:
Make this string translatable like following:

https://github.com/openwisp/openwisp-users/blob/84e7cd145f5243b452c94f14d62ae92e93557483/openwisp_users/multitenancy.py#L35

Reply:
@pandafy sure. One more thing, as you can see I wrote a test for this, but after implementing these changes, I can't test this feature as there is no model with optional organization field in this project.